libs/librepcb/library/pkg/footprintpad.h

```diff
@@ -78,7 +80,13 @@ class FootprintPad final : public SerializableObject {
                Shape shape, const PositiveLength& width,
                const PositiveLength& height,
                const UnsignedLength& drillDiameter, BoardSide side) noexcept;
+  FootprintPad(const Uuid& padUuid, const Point& pos, const Angle& rot,
+               Shape shape, const PositiveLength& width,
+               const PositiveLength&          height,
+               const tl::optional<DrillSize>& drillSize,
+               BoardSide                      side) noexcept;
```

Comment:
Same here about "convenience" constructors: Please don't add an additional constructor, but just adjust the existing constructor. The old constructor does not provide any advantage.

Reply:
I removed everything I'd consider a "convenience constructor" and migrated other code to use the remaining ones